modules/mod_ginger_rdf/models/m_rdf.erl

```diff
@@ -20,6 +20,8 @@
 
 m_find_value(#rdf_resource{} = Rdf, #m{value = undefined} = M, _Context) ->
     M#m{value = Rdf};
+m_find_value(Id, #m{value = undefined}, Context) when is_integer(Id) ->
+    mochijson2:encode(ginger_json_ld:serialize(to_triples(Id, Context)));
```

Comment:
Sorry missed the guard. Tested the branch locally on JHM seems to work. I'll merge it now.

BTW. I do think m_value is called for when you do `{{ m.rdf[id] }}` if you return an m# record.